apps/prairielearn/elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -391,9 +391,7 @@ def grade(element_html, data):
     if comparison == "relabs":
         rtol = pl.get_float_attrib(element, "rtol", RTOL_DEFAULT)
         atol = pl.get_float_attrib(element, "atol", ATOL_DEFAULT)
-    elif comparison == "sigfig":
-        digits = pl.get_integer_attrib(element, "digits", DIGITS_DEFAULT)
-    elif comparison == "decdig":
+    elif comparison == "sigfig" or comparison == "decdig":
```

Comment:
Another potential change:

```suggestion
    elif comparison in {"sigfig", "decdig"}:
```

Reply:
This would be resolved by the `pylint` ruff rule: https://docs.astral.sh/ruff/rules/repeated-equality-comparison/#repeated-equality-comparison-plr1714, which is not split up into a separate PR yet (still in #11056 )